src/components/BaseComponents.js

```diff
@@ -101,6 +101,19 @@ export const Validations = {
         return parseFloat(value) > maxValue
             ? `Loan amount is greater than currently available maximum of ${maxValue} A-EUR`
             : undefined;
+    },
+    maxExchangeValue: value => {
```

Comment:
This is unnecessary, easier to check if `value` is greater than `state.simpleResult.filledTokens`.